web/docs/setup/deploy/docker-compose.md

```diff
@@ -60,22 +65,24 @@ Docker Compose:
 # Load both VAST and Zeek, and the import that sits between the two.
```

Comment:
Unrelated to this line, but if you look at https://vast.io/docs/setup/deploy/docker-compose then you can see that the :::info block above is not properly closed out and extends into this text. Can you fix this as well?
Unrelated to this line, but if you look at https://vast.io/docs/setup/deploy/docker-compose then you can see that the :::info block above is not properly closed out and extends into this text. Can you fix this as well?

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/4488655/204784615-0cb4fa34-cbd7-4310-92c0-f41a0ced315c.png">

Reply:
On it. I am also changing the sentence 
```
...try using `docker-compose` instead—although we don't test it explicitly, most commands
should work that way as well.
```
to 
```
...try using `docker-compose` instead. We don't test our script explicitly with this older
version, so although most commands should work, we recommand upgrading to a
recent version of Docker and Compose.
```